pkgs/by-name/pi/picocrypt/package.nix

```
@@ -0,0 +1,78 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  copyDesktopItems,
+  makeDesktopItem,
+
+  xorg,
+  libGL,
+  gtk3,
+  pkg-config,
+}:
+
+buildGoModule rec {
+  pname = "picocrypt";
+  version = "1.34";
+
+  src = fetchFromGitHub {
+    owner = "Picocrypt";
+    repo = "Picocrypt";
+    rev = version;
+    hash = "sha256-TO72s8v0cpyKjvi0b74vux3+VzTfW540Drtr2bD5xVw=";
+  };
+
+  sourceRoot = "${src.name}/src";
+
+  vendorHash = "sha256-W982HiosXvDadMJJ0wP6AsalQ/uxklSbbmFp26XQEhM=";
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  buildInputs = [
+    xorg.libX11.dev
+    xorg.libXcursor.dev
+    xorg.libXrandr.dev
+    xorg.libXi.dev
+    xorg.libXinerama.dev
+    xorg.libXxf86vm
+    libGL.dev
+    gtk3
+  ];
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    pkg-config
```

```suggestion
    pkg-config
    wrapGAppsHook3
```

You never know when will others request you to add it. Just add it in case of gtk3/4.